CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 -   Checkbox: add `disabled` attribute to the `input` element
+-   New component in React `GridColumn`
```

Comment:
Need to move this under "Unreleased". It's probably because you rebased once 3.0.7 was merged and git does not see this as a conflict